package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "spectral-ruleset",
```

Comment:
I think we should prefix that with `@baloise`? (see also https://github.com/baloise/design-system/blob/master/packages/components/package.json#L2)
Maybe @hirsch88 can give some input here?

Reply:
I added baloise- prefix. I can also use @baloise

Reply:
done